worksheet_C/readme.MD

```diff
@@ -0,0 +1,10 @@
+Worksheet C  
+A) The algorithm checks if there is a duplicate letter in the list by going through the list in order to see if a letter is repeated, if a letter has a duplicate the result will be true. 
```

Comment:
✅ (technically it's not specified that it's a list of letters, so "element" or "item" would be more general)